server/routes/index.js

```diff
@@ -1,9 +1,9 @@
 const express = require('express');
 
+const { servicesHandler } = require('../controller');
+
 const router = express.Router();
 
-router.get('/hi', (req, res) => {
-  res.send('hi');
-});
+router.get('/services', servicesHandler);
```

Comment:
please create a common.js file in the routes folder and add your route to it.

common.js
```js
const commonRouter = require('express').Router();

const {
  serviceHandler
} = require('../controllers');

commonRouter.get('/services', serviceHandler);

module.exports = commonRouter;
```
then import it into the index.js
```js
const commonRouter = require('./common');

router.use(commonRouter);
```